multinode/bootstrapping.md

```diff
@@ -0,0 +1,191 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up multi-node Timescale is achieved by adding data nodes to
```

Comment:
The product is called TimescaleDB and `multi-node` seems to be tautology to `scaling up`:
```suggestion
Scaling up TimescaleDB is achieved by adding data nodes to
```